### PR TITLE
Add requirement for ephemeral storage

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: content/kotsadm/installing/system-requirements.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ IP addresses for these services can be found in [replicatedhq/ips](https://githu
 
 No outbound internet access is required for airgapped installations.
 
+## Airgapped Installs
+
+When application airgap bundle is uploaded through the Admin Console's UI, application images will be extracted and pushed to the registry from inside the Admin Console's pod.
+This requires extra ephemeral storage available on the node.
+The recommended amount of storage should be no less then the total size of application's images.
```
Comment (dexhorthy) [inline content/kotsadm/installing/system-requirements.md:55]:
```suggestion
The recommended amount of storage should be no less than the total size of application's images.
```

it's all I'm good for anymore 😂

Comment (divolgin) [reply content/kotsadm/installing/system-requirements.md:55]: omg

Comment (marccampbell) [reply content/kotsadm/installing/system-requirements.md]: Is this accurate? Isn't the ephemeral storage requirement actually the size of the Airgap bundle archive in addition to the size of the applications's images uncompressed? I'm curious to understand the details here.

**Comment by marccampbell** [comment]
As a vendor shopping to an enterprise, how can I set that requirement? If I'm an enterprise, what does "ephemeral storage" mean in this case? Is it a Kubernetes manifest attribute, or the total node available storage? Can we be more explicit in the requirement here? I'm concerned that the phrasing here might not be clear enough for someone.

**Comment by AmberAlston** [comment]
@divolgin should we close this out?

**Comment by AmberAlston** [comment]
@divolgin Let's close this out if it's no longer necessary. It's getting to be an old PR

**Comment by AmberAlston** [comment]
@divolgin Can we close this given its age?

**Comment by AmberAlston** [comment]
Closed due to age. We can bring it back if it's still relevant.

**Pull request closed without merging**